### PR TITLE
Support for --with-oshrun-launcher configure flag

### PR DESCRIPTION
**Pull request by richardnorth3**
Signed-off-by: richardnorth3 <richard.north.iii@intel.com>

### CODE REVIEW

File: src/Makefile.am [hunk 1]
```diff
@@ -156,7 +156,9 @@ do_subst = sed -e 's|[@]PERL[@]|$(PERL)|g' \
                -e 's|[@]WRAPPER_COMPILER_INCLUDEDIR[@]|$(includedir)|g' \
                -e 's|[@]WRAPPER_COMPILER_LIBDIR[@]|$(libdir)|g' \
                -e 's|[@]WRAPPER_COMPILER_EXTRA_LDFLAGS[@]|$(WRAPPER_COMPILER_EXTRA_LDFLAGS)|g' \
-               -e 's|[@]WRAPPER_COMPILER_EXTRA_LIBS[@]|$(WRAPPER_COMPILER_EXTRA_LIBS)|g'
+               -e 's|[@]WRAPPER_COMPILER_EXTRA_LIBS[@]|$(WRAPPER_COMPILER_EXTRA_LIBS)|g'
```
Comment (davidozog) [inline src/Makefile.am]: Looks like there might be a spacing issue or something here?  I don't see any obvious change to this line...

Comment (richardnorth3) [inline src/Makefile.am]: Ah, I think I see it. Let me push the fix through

**Comment by jdinan** [comment]
bot:lanl:retest

**Comment by jdinan** [comment]
Looks good!

**Comment by davidozog** [comment]
Note: this merge probably closes #420.

**Pull request merged**